reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/SwitchIfEmpty.kt

```diff
@@ -0,0 +1,52 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.base.ErrorCallback
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+import com.badoo.reaktive.utils.atomicreference.AtomicReference
+
+fun <T> Observable<T>.switchIfEmpty(otherObservable: Observable<T>): Observable<T> =
+    switchIfEmpty { otherObservable }
+
+fun <T> Observable<T>.switchIfEmpty(otherObservable: () -> Observable<T>): Observable<T> =
+    observableUnsafe { observer ->
+        val disposableWrapper = DisposableWrapper()
+        observer.onSubscribe(disposableWrapper)
```

Comment:
When you convert it to `emitter` then just call `emitter.setDisposable` instead of `observer.onSusbcribe`